mojave/jobs/job_types/wasteland/wastelander.dm

```diff
@@ -15,19 +15,73 @@
 	name = "_Wastelander"
 	jobtype = /datum/job/ms13/wasteland/wastelander
 
-	head = /obj/item/clothing/head/collectable/petehat
-
 /datum/outfit/job/ms13/wasteland/wastelander/pre_equip(mob/living/carbon/human/H)
 	..()
+	if(prob(50))
+		head = pick(
+		/obj/item/clothing/head/helmet/ms13/cowboy/black,\
+		/obj/item/clothing/head/helmet/ms13/utilitycap,\
+		/obj/item/clothing/head/helmet/ms13/baseball,\
+		/obj/item/clothing/head/helmet/ms13/cowboy/padded/ushanka)
+	else
+		head = null
+
+	if(prob(25))
+		mask = pick(
+		/obj/item/clothing/mask/ms13/facewrap,\
+		/obj/item/clothing/mask/ms13/facewrap/red,\
+		/obj/item/clothing/mask/ms13/facewrap/black,\
+		/obj/item/clothing/mask/ms13/facewrap/blue,\
+        /obj/item/clothing/mask/gas,\
+        /obj/item/clothing/mask/bandana/black,\
+        /obj/item/clothing/mask/balaclava)
+	else
+		mask = null
+
+	if(prob(35))
```

Comment:
Every role should have a backpack. The cosmetic appearance of said backpack can be up in the air, but every role should have one.

Reply:
kinda cucked but ok